### DISCUSSION TITLE
邮件发不出去

### DISCUSSION THREAD

**Discussion by BitterColaaaaa** [Jinnrry/PMail#227] [Q&A]
报错如下：
[info][2024-11-16 15:10:17][ac11000367384559d09c0001fca2bbb0][/work/controllers/email/send.go:58]发送邮件
[error][2024-11-16 15:10:20][ac11000367384559d09c0001fca2bbb0][/work/utils/send/send.go:213][0xc0001b5d00] 邮件投递失败dial tcp 94.23.206.89:25: i/o timeout
[error][2024-11-16 15:10:20][/work/hooks/framework/framework.go:123]params error json: cannot unmarshal object into Go struct field HookDTO.ErrMap of type error
[error][2024-11-16 15:10:20][/work/hooks/framework/framework.go:123]params error json: cannot unmarshal object into Go struct field HookDTO.ErrMap of type error
[error][2024-11-16 15:10:20][/work/hooks/framework/framework.go:123]params error json: cannot unmarshal object into Go struct field HookDTO.ErrMap of type error

**Comment by Jinnrry** [comment]
timeout 属于网络问题，我也没法解决。排查服务器25端口是不是被服务商封了

**Reply by BitterColaaaaa** [reply]
换了台服务器专门搞pmail，已经弄好了，谢谢
